LDAR_Sim/src/methods/sensors/AVO.py

```diff
@@ -0,0 +1,22 @@
+import numpy as np
+
+
+def detect_emissions(self, site, leaks_present, equipment_rates, site_true_rate, venting):
```

Comment:
I think we need to make this so that each leak is either detectable or not and not a random chance each time?

Reply:
Fixed, If the random binomial returns false, the a flag is_op_detectable is set to false and the leak is not checked again by the Operator agent.